odoo/addons/base/data/res.country.state.csv

```diff
@@ -1406,4 +1406,3 @@ state_ec_21,base.ec,Sucumbios,21
 state_ec_22,base.ec,Orellana,22
```

Comment:
this diff should not be here

Reply:
Please can explain a little bit more this issue, maybe states information can't be loaded by this module?

Reply:
Don't know how can I resolve this issue, maybe rebase master branch?